### ISSUE TITLE
Notebook Find width cutting off "of x results" portion of widget -- Chinese Simplified

### ISSUE THREAD

**Issue by ease-zh** [microsoft/vscode#226592]
Does this issue occur when all extensions are disabled?: No

- VS Code Version: 1.92.2 (user setup)
- OS Version:  Windows_NT x64 10.0.22631
Steps to Reproduce:

1.  Create a .ipynb file and a .py file, then type any thing in them.
2. Search in the py file,  we can see the full information about the search result.
![image](https://github.com/user-attachments/assets/7f50fee4-c972-4435-bf28-73bdf738881f)
3.  Search in the ipynb file, it can not show the total num of matches. 
![image](https://github.com/user-attachments/assets/6267f00d-6df8-413a-be76-c43cbc8c77d2)

**Comment by Yoyokrazy** [comment]
dupes all seem to be in the same language, likely an issue with the specific language translation and the size of the right side container. 

Could you please tell me which specific display language you are currently using?

**Comment by Shi-YuZhang** [comment]
Chinese (Simplified) here

**Comment by ease-zh** [comment]
> dupes all seem to be in the same language, likely an issue with the specific language translation and the size of the right side container.
> 
> Could you please tell me which specific display language you are currently using?

You are right, when I use Chinese (Simplified) the problem occurs, and once I switch to English, it just works fine.

For english, the result is just "x of xxx"，which takes smaller space. For chinese, the result contain more characters, "第x项，共XXX项".  Maybe the search result space is not suited for these many characters.

**Comment by Yoyokrazy** [comment]
Thanks for the quick replies, yeah sounds like a container size constraint. Not sure why that secondary action menu would have a max-width property, but it seems like it has something to do with that. Will take a look at this when I get some time in the future, thanks guys.

**Comment by Yoyokrazy** [comment]
Fixed by https://github.com/microsoft/vscode/pull/250738
